infra/upload_images/container_ocr/handler/ocr_processor.py

```
@@ -176,6 +194,182 @@ def _process_refinement_job(
+        This method just creates DynamoDB entities from the pre-processed data.
+        """
+        image_id = ocr_job.image_id
+        current_time = datetime.now(timezone.utc)
+
+        # Get image type from classification
+        classification = ocr_json.get("classification", {})
+        image_type_str = classification.get("image_type", "NATIVE").upper()
+        try:
+            image_type = ImageType[image_type_str]
+        except KeyError:
+            image_type = ImageType.NATIVE
```

Ruff (0.14.10)

Local variable `image_type` is assigned to but never used

Remove assignment to unused variable `image_type`

(F841)